src/compiler/optimize/optimize-module.ts

```diff
@@ -170,7 +170,7 @@ export const prepareModule = async (
     if (tsResults.sourceMapText) {
       // need to merge sourcemaps at this point
       const mergeMap = sourceMapMerge(
-        (minifyOpts.sourceMap as SourceMapOptions).content as SourceMap,
+        (minifyOpts.sourceMap as SourceMapOptions)?.content as SourceMap,
```

Comment:
`minifyOpts.sourceMap` is typed as  `sourceMap?: boolean | SourceMapOptions;`. This doesn't fix the type assertion usage (that's for a later day), but it does prevent accessing a field on an `undefined` value